core/client/fs/src/main/java/alluxio/client/file/cache/store/MemoryPageStore.java

```diff
@@ -94,4 +107,52 @@ public void close() {
   public void reset() {
     mPageStoreMap.clear();
   }
+
+  private static class MemPage {
+    private final byte[] mPage;
+    private int mPageLength;
+
+    public MemPage(byte[] page, int pageLength) {
+      mPage = page;
+      mPageLength = pageLength;
+    }
+
+    public byte[] getPage() {
+      return mPage;
+    }
+
+    public int getPageLength() {
+      return mPageLength;
+    }
+
+    public void setPageLength(int pageLength) {
+      mPageLength = pageLength;
```

Comment:
Check `0 <= pageLength <= mPage.length`?

Reply:
> Check `0 <= pageLength <= mPage.length`?

Agree. This should be checked.

Reply:
will fix this in another PR, let's merge this one for release